samples/healthcheck/pom.xml

```diff
@@ -39,6 +39,11 @@
                   <cmd>curl -f http://localhost/ || exit 1</cmd>
                 </healthCheck>
               </build>
+              <run>
+                <wait>
+                  <healthy>true</healthy>
```

Comment:
I wonder what `<healthy>false</healthy>` would imply ;-) 

I know, Maven doesn't allow empty tags, so that's probably fine.